praxis-notes-app/src/ai/providers/get-ai-provider-name-from-model-name.provider.ts

```
@@ -0,0 +1,16 @@
+import { Function } from '@errors/types';
```

Biome (1.9.4)

[error] Do not shadow the global "Function" property.

Consider renaming this variable. It's easy to confuse the origin of variables when they're named after a known global.

(lint/suspicious/noShadowRestrictedNames)